docs/guides/qedma-qesem.ipynb

```diff
@@ -143,143 +167,171 @@
    "outputs": [],
    "source": [
     "job = qesem_function.run(\n",
-    "    action = \"execute\",\n",
-    "    instance = \"hub/group/project\",\n",
+    "    instance = \"hub1/group1/project1\",\n",
     "    pubs = [\n",
     "        (\n",
     "            circ,\n",
-    "            [obs1,obs2,obs3]\n",
+    "            [avg_magnetization, other_observable],\n",
+    "            None,\n",
+    "            0.03\n",
     "        )\n",
     "    ],\n",
-    "    precision = 0.03,\n",
-    "    run_options = {\n",
-    "        \"backend_name\": \"ibm_brisbane\",\n",
-    "    }\n",
+    "    backend_name = \"ibm_brisbane\",\n",
+    "    options = {\n",
+    "        \"estimate_time_only\": False,\n",
+    "    },\n",
     ")"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "399b0deb",
+   "id": "dfe99043",
    "metadata": {},
    "source": [
-    "<Admonition type=\"note\">\n",
-    "- The `precision` parameter signifies the acceptable error on the expectation values of the observables in absolute value.  Namely, the QPU runtime for mitigation will be determined to provide output values for all the observables of interest that fall within a `1σ` confidence interval of the target precision. If multiple observables are provided, the mitigation will run until the target precision is reached for each of the input observables.\n",
+    "You can use the familiar Qiskit Serverless APIs to check your Qiskit Function workload's status or return results:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "856fe992",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(job.status())\n",
+    "result = job.result()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8b901258",
+   "metadata": {},
+   "source": [
+    "## Function parameters\n",
+    "\n",
+    "\n",
+    "| Name |  Type | Description | Required | Default |  Example |\n",
+    "| -----| ------| ------------| -------- | ------- | -------- |\n",
+    "| `pubs` | [EstimatorPubLike](https://github.com/Qiskit/qiskit/blob/5121a0fda8d4e49d47d16a28e1d21046a09d008b/qiskit/primitives/containers/estimator_pub.py#L202-L207) |This is the main input. The `Pub` contains 2-4 elements: a circuit, one or more observables, 0 or a single set of parameter values, and an optional precision. If a precision was not specified, then the `default_precision` from `options` will be used|  Yes|  N/A |  `[(circuit, [obs1,obs2,obs3], parameter_values, 0.03)]`  |\n",
```

Comment:
The link to this should be `/guides/primitive-input-output`

Reply:
done